ark/utils/spatial_lda_utils.py

```diff
@@ -1,63 +1,116 @@
-import os
-import numpy as np
 import pandas as pd
 
-def check_format_cell_table_args(cell_table, markers, clusters, fovs):
+from ark.settings import BASE_COLS
 
-    """
-    Checks the input arguments of the format_cell_table() function.
+
+def check_format_cell_table_args(cell_table, markers, clusters):
+    """Checks the input arguments of the format_cell_table() function.
 
     Args:
         cell_table (pd.DataFrame):
-            A pandas DataFrame containing the columns of cell marker frequencies and/or cluster ids.
+            A pandas DataFrame containing the columns of cell marker
+            frequencies and/or cluster ids.
         markers (list, str):
             A list of strings corresponding to marker names.
         clusters (list, int):
             A list of integers corresponding to cluster ids.
-        fovs (str, int):
-            One of either "all_fovs" indicating all field of views are to be kept, or a list of integers corresponding
-            to the index of each field of view which should be kept.
 
-    Returns:
-        None
     """
 
-    base_cols = [
-        "point",
-        "label",
-        "cell_size",
-        "centroid-0",
-        "centroid-1",
-        "pixelfreq_hclust_cap",
-        "name"
-    ]
-
     # Check cell table
-    if not isinstance(cell_table, pd.DataFrame):
-        raise ValueError("cell_table must be a pd.DataFrame")
-
-    if not all([True for x in base_cols if x in cell_table.columns]):
-        raise ValueError("cell table must contain the following columns:{}".format(base_cols))
+    if not all([x in cell_table.columns for x in BASE_COLS]):
+        raise ValueError(
+            "cell table must contain the following columns:{}".format(
+                BASE_COLS))
 
     # Check markers/clusters
     if all([markers is None, clusters is None]):
```

Comment:
See Alex's comment in original PR about more pythonic phrasing for some of these tests